eng/pipelines/macos.yml

```diff
@@ -79,6 +79,7 @@ jobs:
       buildExtraArguments: ''
 
       variables:
+        - _helixPreCommands: "chmod a+x $HELIX_CORRELATION_PAYLOAD/InstallRootCertificate.sh;sudo -E -n $HELIX_CORRELATION_PAYLOAD/InstallRootCertificate.sh --service-host $(ServiceUri) --cert-file /tmp/wcfrootca.crt"
```

Comment:
I still think that you should probably add the cert as a correlation payload a reference it that way because I have a feeling that there was something else that managed getting it on the machine and putting it in temp